### PR TITLE
In 145765803 implement datetime picker

### PR DESCRIPTION
**Pull request by AmanShurafa**

### CODE REVIEW

File: spec/screening/required_field_spec.rb [hunk 1]
```diff
@@ -123,7 +123,7 @@
           within(table_rows[0]) do
             row0_id = find('input[id^="allegations_"]')[:id]
             find_field(row0_id).send_keys(:backspace)
-            fill_in_react_select(row0_id, with: (allegation).to_s)
+            fill_in_react_select(row0_id, with: allegation.to_s)
```
Comment (jgedeon-osi) [inline spec/screening/required_field_spec.rb]: Might as well take off the to_s, allegation is already a string at this point.

File: spec/screening/required_field_spec.rb [hunk 2]
```diff
@@ -146,7 +146,7 @@
...
```
Comment (jgedeon-osi) [inline spec/screening/required_field_spec.rb]: Allegation is already a string at this point, I don't think we need to call to_s.

File: spec/screening/screening_info_spec.rb [hunk 3]
```diff
@@ -76,15 +76,16 @@
       expect(page).to have_select('Communication Method',
                                   selected: screen_info_init[:comm])
       fill_in('Assigned Social Worker', with: screen_info_chng1[:worker])
-      fill_in_datepicker('Screening Start Date/Time', with: screen_info_chng1[:sdate])
+      mouse_select_datepicker('#started_at', 12) # Date.parse(screen_info_chng1[:sdate]).day = 12
```
Comment (jgedeon-osi) [inline spec/screening/screening_info_spec.rb]: Can we not still use the info in screen_info_chng1? As apposed to hard coding 12 in here?

**Comment by aaronOSI** [comment]
Looks good to me

File: spec/screening/screening_info_spec.rb [hunk 4]
```diff
@@ -76,15 +76,16 @@
...
+      mouse_select_timepicker('#started_at', '3:30 PM')
```
Comment (jgedeon-osi) [inline spec/screening/screening_info_spec.rb]: Can we not still use the info in screen_info_chng1? It has a time of 1:00 pm in it. I prefer that to hard coded numbers.

**Pull request merged**